cl/opinion_page/templates/includes/authorities_section.html

```diff
@@ -0,0 +1,33 @@
+{% load humanize %}
+{% load extras %}
+{% load humanize %}
+{% load static %}
+{% load text_filters %}
+
+    {# Show cases this case cites #}
+        {% if authorities_context.total_authorities_count > 0  %}
+            <div id="authorities" class="sidebar-section">
+                <h3><span>Authorities ({{ authorities_context.total_authorities_count|intcomma }})</span></h3>
+                <p class="bottom">This opinion cites:</p>
+                <ul>
+                    {% for authority in authorities_context.top_authorities %}
+                        <li>
+                            <a href="{{ authority.url }}">
+                                {{ authority.caption|safe|truncatewords:10|v_wrapper }}
+                            </a>
+                            {% if authority.count%}
+                              <span class="small gray">({{ authority.count}}&nbsp;time{{ authority.count|pluralize }})</span>
+                            {% endif %}
+                        </li>
+                    {% endfor %}
+                </ul>
+                {% if authorities_context.view_all_url|length > 0 %}
```

Comment:
I'm surprised this wouldn't work: 

```suggestion
                {% if authorities_context.view_all_url %}
```